core/java/com/google/instrumentation/stats/ViewDescriptor.java

```diff
@@ -81,6 +79,43 @@ private ViewDescriptor(
   }
 
   /**
+   * Visitor for the ViewDescriptor type.
```

Comment:
I suppose I'm wondering if we can just not add boiler-plate javadoc when there's nothing notable to say